apps/src/templates/teacherDashboard/TeacherDashboard.jsx

```diff
@@ -154,13 +168,28 @@ function TeacherDashboard({
         />
         {coursesWithProgress.length === 0 && (
           <Route
-            component={() =>
-              applyV1TeacherDashboardWidth(
-                <div className={dashboardStyles.text}>
-                  <SafeMarkdown markdown={i18n.noProgressSection()} />
-                </div>
-              )
-            }
+            component={props => (
+              <div className={dashboardStyles.emptyClassroomDiv}>
+                {location.pathname === TeacherDashboardPath.progress && (
```

Comment:
Could you explain why we need the different className for progress and non-progress pages? Shouldn't it looks the same in all tabs?

Reply:
If we do need this change, could we move it to a helper function in this file to deduplicate with the above OR put the logic into EmptySection

Reply:
Great question... they are different in order to take into account the `Progress` page has a `Progress` header at the top and the others don't.  I suppose this is a design choice I made.  Without additional margin on the non-Progress pages the image moves up the page when switching between tabs.  From a design perspective, I liked the consistency of the location of the image on the page.  I also recognize there might be a more elegant solution to this problem using CSS...